source/data/iCommands.rst

```diff
@@ -0,0 +1,370 @@
+.. _iCommands:
+
+iCommands
+===============
+
+iCommands is one of the client-side communication with iRODS server to provide users with data management and metadata management functions to do any data-related actions. In short, iCommands is an Unix utility that gives users a command-line interface.
+There are more than 50 iCommands. A regular user however may use only a few of them for his/her daily needs. iRODS offers other user interfaces but the underlying point is that iCommands is the most powerful and easy to use interface to iRODS.
+
+All iCommands accept standard common line options (e.g., -a for all, -h for help) that gives more capabilities to the commands. To see a subset of these options and to know the details of any iCommand, you can follow the below specified options:
+
+- You can visit the `iCommands documentation <https://docs.irods.org/4.2.6/icommands/user/>`__
+ 
+- You can use the ``–h`` option with the command (e.g., ``iput –h``)
+  
+- You can use the ``ihelp`` command with the argument that you would like to learn more about (e.g., ``ihelp iput``).
+  
+Please keep in mind some iCommands don't work with tab press auto-complete. Also remember that folders in iRODS are called 'Collections' and files are called 'Objects' or 'Data Objects'.
+
+The following sections illustrate the usage of some iCommands organized on the following categories: "Informative commands", "Working with Collections", "Data upload and download", "Structuring data", "Access Control" and "Handling metadata".
+
+Installing iCommands locally
+----------------------------
+iCommands is installed on the VSC Tier-1 and some of the Tier-2 clusters. As it is a client to any iRODS system, it can also be used from any local computer after installing it there.
```

Comment:
Found this only now: iCommands are only installed on our Tier-1 cluster, not the one of UGent.

Reply:
@kwaegema ^

Reply:
Changed to KU Leuven Tier-1.